markdown/dev/reference/api/part/config/stack/en.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Assigning parts to stacks
+---
+
+The optional `stack` property assigns the part to a specific
+[Stack](/reference/api/stack).
+
+It holds either a string with the stack name or a function which
+returns the stack name:
+
+```js
+const part = {
+  name: 'example.front',
+  stack: 'example.combined',
+}
+```
+
+```js
+const part = {
+  name: 'example.back',
+  stack: ({options}) => options.stack ? 'example.combined' : 'example.back',
+}
+```
+
+## Notes
+
+If `stack` is present, the part is assigned to the stack with the
+specified name.
+If multiple parts are assigned to the same stack, they will overlap
+in drafting and printing layouts.
+This is because parts in the stack are drafted within the same stack
+space.
+
+Otherwise, if the `stack` property is not present, the default behavior
+is to assign the part to its own stack.
```

Comment:
Otherwise, if the `stack` property is not present, the default behavior
is to use the part's name as its stack name.

Reply:
Added.